README.md

```diff
@@ -103,6 +99,20 @@ nodes being offline for more than than maxAge days are considered "lost".
 Max level to be applied by clicking a node or open a node. With value `18` near by buildings and streets should be visible.
 Interesting if one of configured map provider has zoom-level under `18`.
 
+## nodeInfobox
+
+### showContact (bool)
```

Comment:
h3 is to large

Maybe current README needs more consistent between h4 or code syntax.
https://github.com/ffrgb/meshviewer#maplayers-list
https://github.com/ffrgb/meshviewer#fixedcenter-array-optional